data/lua/core/_initial.lua

```diff
@@ -73,9 +73,4 @@ end
 
 unpack = wesnoth.deprecate_api('unpack', 'table.unpack', 3, '1.17', table.unpack)
 math.pow = wesnoth.deprecate_api('math.pow', '^', 3, '1.17', function(a,b) return a ^ b end)
-if wesnoth.kernel_type() == "Game Lua Kernel" then
-	-- wesnoth.wml_actions.music doesn't exist yet at this point, so create a helper function instead.
-	wesnoth.set_music = wesnoth.deprecate_api('wesnoth.set_music', 'wesnoth.music_list', 1, nil, function(cfg)
-		wesnoth.wml_actions.music(cfg)
-	end)
```

Comment:
I'm a little confused about this - if `wesnoth.wml_actions.music` doesn't actually exist, what was this doing?

Reply:
Emphasis on "at this point". It's not that it doesn't exist; it's just loaded later than this file, so _at the point when this file is executed_, it doesn't exist; but by the time anything calls this function, it _will_ exist.